README.md

```diff
@@ -19,12 +19,12 @@ yarn add sapera-components@^0.0.1-development
 import { Button } from "sapera-components";
 ```
 
-`sapera-components` provides an integrated TypeScript experience out of the box.<br/>
-The types for each component is provided (following the pattern `{Component}Props` and can be imported for
-type-checking while developing.
+`sapera-components` provides an integrated TypeScript experience out of the box.<br/>	
+The types for each component is provided (following the pattern `{Component}Props` and can be imported for	
+type-checking while developing.	
 
-```javascript
-import { Button, ButtonProps } from "sapera-components";
+```javascript	
+import { Button, ButtonProps } from "sapera-components";	
```

Comment:
any specific reasons for adding those spaces? for example Silvio's editor (emacs) removes the trailing spaces on every save... so it'll be removed... if you want to add a line after a block, use `<br />` instead of 2 spaces as it's safer and works on every machine with every editor and explictly setting the style

Reply:
No specific reasons. The whitespaces was a mistake when I copied and pasted the description back into the readme after removing it accidentally. I created a Pull Request for this fix [here](https://github.com/infographicsgroup/sapera-components/pull/6). I was unable to push the changes directly into the master branch because it is a protected branch.
Thanks for spotting the issue. My VSCode settings had _Trim Trailing Whitespace_ on saving off.